vuu-ui/packages/vuu-ui-controls/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@salt-ds/core": "1.13.2",
+    "@salt-ds/icons": "1.9.1",
```

Comment:
I don't think we need to specify this as a dependency. It's already a dependency of salt core.
We never want to user these directly ourselves. 
In fact, these icons are a bit of a headache, we don't really want to use Salt icons, we would rather use our own. Thats easy with the way we specify icons (in css) but not easy with salt componnets as they ship them as react components.

In our theme CSS you will finsd this

```
 .saltIcon {
    display: none;
} 
```